descriptors/ModuleDescriptor-template.json

```diff
@@ -113,7 +117,8 @@
             "configuration.entries.collection.get",
             "inventory-storage.holdings-note-types.collection.get",
             "inventory-storage.item-note-types.collection.get",
-            "inventory-storage.call-number-types.collection.get"
+            "inventory-storage.call-number-types.collection.get",
+            "inventory-storage.instance-bulk.ids.get"
```

Comment:
/this permission need to be given to upload endpoint isn't it? That's where you are querying to fetch the UUIDs

Reply:
yes, it's permission required to retrieve UUIDs by /instance-bulk/ids

Reply:
you added it in the incorrect place Illia, Please add it to the upload endpoint